### DISCUSSION TITLE
部署完之后似乎会修改yarn.lock文件导致不能git pull

### DISCUSSION THREAD

**Discussion by Miuzarte** [Tsuk1ko/CQ-picfinder-robot#422] [一般讨论]
不知道是不是我的问题，似乎是部署完了之后yarn.lock就会变成modified的状态
![image](https://github.com/Tsuk1ko/cq-picsearcher-bot/assets/66856838/59967100-21f1-4fef-8a5d-82035651bd5b)
然后导致没法更新

**Comment by Tsuk1ko** [comment]
变更内容是什么，截图看看

**Reply by Miuzarte** [reply]
> 变更内容是什么，截图看看

等我重新yarn一下，刚给覆盖掉了，~~不过刚刚看了眼好像是变成了一万多行~~（

**Reply by Miuzarte** [reply]
> 变更内容是什么，截图看看

不对 想起来了，我好像是因为嫖别人的机子 没root，用的npm i --production

**Reply by Miuzarte** [reply]
> 变更内容是什么，截图看看

[yarn.lock.zip](https://github.com/Tsuk1ko/cq-picsearcher-bot/files/11442865/yarn.lock.zip)

**Reply by Miuzarte** [reply]
> 变更内容是什么，截图看看

还有这个也更改了一行
![image](https://github.com/Tsuk1ko/cq-picsearcher-bot/assets/66856838/f95bcb4f-a6a5-4102-885c-e69cc1478f84)

**Reply by Tsuk1ko** [reply]
az，可能是因为 npm v8 开始支持 yarn.lock 了……所以就会用它来安装还把它改了

**Reply by Tsuk1ko** [reply]
你试试先还原下，用 `npm i --production --no-save` 看看行不行

**Reply by Tsuk1ko** [reply]
不过这样就算行的话可能每次 install 都会很慢，我下个版本直接内置 yarn 得了（顺便还能体验一下 yarn v2

**Reply by Miuzarte** [reply]
> 你试试先还原下，用 `npm i --production --no-save` 看看行不行

网络有点寄
![image](https://github.com/Tsuk1ko/cq-picsearcher-bot/assets/66856838/7cd774c2-2dd4-4460-8526-3ed55b26761b)

**Comment by Tsuk1ko** [comment]
算了，想了想用 yarn berry 会导致很多麻烦的问题

干脆把 lockfile 删了，这样就不会有冲突问题
